Source/WebCore/html/HTMLScriptElement.cpp

```diff
@@ -98,6 +109,41 @@ void HTMLScriptElement::setText(String&& value)
     setTextContent(WTFMove(value));
 }
 
+DOMTokenList& HTMLScriptElement::blocking()
+{
+    if (!m_blockingList) {
+        m_blockingList = makeUniqueWithoutRefCountedCheck<DOMTokenList>(*this, HTMLNames::blockingAttr, [](Document&, StringView token) {
+            if (equalLettersIgnoringASCIICase(token, "render"_s))
+                return true;
+            return false;
+        });
+    }
+    return *m_blockingList;
+}
+
+// https://html.spec.whatwg.org/multipage/scripting.html#script-processing-model:implicitly-potentially-render-blocking
+bool HTMLScriptElement::isImplicitlyPotentiallyRenderBlocking() const
+{
+    return scriptType() == ScriptType::Classic && isParserInserted() == ParserInserted::Yes && !hasDeferAttribute() && !hasAsyncAttribute();
+}
+
+void HTMLScriptElement::potentiallyBlockRendering()
```

Comment:
Link to: https://html.spec.whatwg.org/multipage/urls-and-fetching.html#potentially-render-blocking ?